transport/x/grpc/inbound.go

```diff
@@ -134,45 +135,42 @@ func (i *Inbound) getServiceDescs() ([]*grpc.ServiceDesc, error) {
 	if len(procedures) == 0 {
 		return nil, errRouterHasNoProcedures
 	}
-	grpcServiceNameToServiceDesc := make(map[string]*grpc.ServiceDesc)
+
+	serviceNameToMethodNames := make(map[string]map[string]bool)
```

Comment:
aside - consider factoring this block out into its own func so this method is more readable